bokeh/models/tests/test_transforms.py

```diff
@@ -0,0 +1,133 @@
+from textwrap import dedent
+
+import pytest
+
+from bokeh.models import CustomJSTransform, Slider
+
+flexx = pytest.importorskip("flexx")
+
+def test_customjstransform_from_py_func_no_args():
+
+    def cosine():
+        from flexx.pyscript import window
+        return window.Math.cos(x) # noqa
+
+    def v_cosine():
+        from flexx.pyscript import window
+        return [window.Math.cos(x) for x in xs] # noqa
```

Comment:
Something like that is mentioned in the docs (https://flexx.readthedocs.io/en/stable/pyscript/api.html#Math), but they may be out of date. I can't find the `Math` stub in 0.4.1